base/reflection.jl

```diff
@@ -450,6 +457,19 @@ function functionloc(f)
     functionloc(first(mt))
 end
 
+"""
+    function_module(f) -> Module
```

Comment:
the file is not autogenerated. only the docstring parts of it that say they are. read contributing.md, there's a section about what you need to do. add the signature to the rst then genstdlib will populate the docstring content